source/index.html.md

```diff
@@ -251,76 +251,566 @@ curl -v --cookie 'JSESSIONID=EXAMPLE_SESSION_ID' \
+  "fundingAmount": 449000000,
+  "fundingString": "449M",
+  "fundingAmountUSD": 501968139,
+  "fundingStringUSD": "502M",
   "fundingRangeID": 9,
   "fundingRange": ">250M",
-  "lastRoundDate": "2022-07-14T00:00:00.000+00:00",
-  "aggregationDate": "2022-05-23T14:09:44.703+00:00",
-  "acquisitionDate": "2021-05-25T00:00:00.000+00:00",
+  "lastRoundDate": "2023-08-31T06:52:00.000+00:00",
   "sustainabilityMetric": 0.9335,
   "foundedDate": 2010,
+  "georowID": 814818,
   "countryID": 80,
-  "cityID": 34891,
-  "admin3ID": 32298,
-  "admin4ID": 74793,
-  "continentID": 3,
   "country": "Germany",
   "city": "Dresden",
-  "admin3": "Kreisfreie Stadt Dresden",
-  "admin4": "Dresden",
   "continent": "Europe",
   "email": "info@sunfire.de",
   "phone": "+49 351 8967970",
-  "sizeID": 2,
-  "size": "11 - 50",
+  "sizeID": 5,
+  "size": "201 - 500",
   "stageID": 4,
   "stage": "Scaling",
   "linkedinURL": "https://www.linkedin.com/company/sunfire-gmbh/",
   "twitterURL": "https://twitter.com/sunfire_dresden/",
-  "facebookURL": "https://facebook.com/1382355191983196/",
+  "facebookURL": "https://facebook.com/sunfire.dresden/",
   "directURL": "sunfire-668",
   "sustainabilityMetricID": 4,
-  "lastRoundType": "Late VC",
+  "lastRoundAmount": 169000000,
+  "lastRoundAmountUSD": 184431559,
+  "lastRoundAmountString": "169M",
+  "lastRoundAmountStringUSD": "184M",
+  "lastRoundType": "Grant",
   "tags": [
```

Comment:
for the tags, just keep 2, not all of them

Reply:
fixed